src/components/dev-popover/docs.mdx

```diff
@@ -0,0 +1,43 @@
+---
+componentName: 'DevPopover'
+---
+
+**🚧 DO NOT USE IN PRODUCTION, FOR WORK IN PROGRESS ONLY 🚧**
```

Comment:
Added some docs here since I figured they might be helpful during dev, if there are other features similar to Search that we want to annotate.